src/epc/tofCam660/tofCam660.py

```
@@ -347,10 +347,14 @@ def __init__(
         super().__init__(self.settings, self.device)
         self.memory = Memory.create(0)
 
-    def __del__(self):
+    def close(self):
         self.tcpInterface.close()
         self.udpInterface.close()
 
+
+    def __del__(self):
+        self.close()
```

Because we call close explicitly, should we just remove this?  Is there any issue calling close twice?

---

The short answer is it's fine in the destructor (any exceptions are just ignored).
However, I kept this in to keep it closer to upstream, but I'm now thinking we should just remove it since we really don't want this behaviour.